clickable.json

```diff
@@ -1,4 +1,3 @@
 {
     "template": "cmake",
```

Comment:
@mymike00 This change result in an invalid json  you`ll have to remove the "," at the end of line

Reply:
oh, sorry. that's because I added for myself also another line to enable logs by default